### PR TITLE
Minor: Activity feed improvements part 3

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- Remove `data asset` from feed card header in add and delete case
- Have limit the runs shown in the DQ Feed Card to latest 10
- Change the design for `Deleted` Feed
- Fix breadcrumb and icon for `Persona, Roles, Policies`

image
image

image

https://github.com/open-metadata/OpenMetadata/assets/66266464/6225f38e-df63-4919-922a-c9fadd362765

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardV2/FeedCardBody/TestCaseFeed/TestCaseFeed.tsx [hunk 1]
```diff
@@ -49,7 +49,10 @@ function TestCaseFeed({
           selectedTimeRange={PROFILER_FILTER_RANGE.last7days.title}
           testCaseName={testCaseName}
           testCaseParameterValue={entitySpecificInfo?.parameterValues}
-          testCaseResults={entitySpecificInfo?.testCaseResult ?? []}
+          testCaseResults={(entitySpecificInfo?.testCaseResult ?? []).slice(
+            0,
+            10
+          )}
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardV2/FeedCardBody/TestCaseFeed/TestCaseFeed.tsx:55]: Can we please do the operations outside the jsx?

**Review by Sachin-chaurasiya — APPROVED** [review]

**Comment by Ashish8689** [comment]
@aniketkatkar97  changes done as per commets

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardV2/FeedCardBody/TestCaseFeed/TestCaseFeed.tsx [hunk 2]
```diff
@@ -49,7 +54,7 @@ function TestCaseFeed({
           selectedTimeRange={PROFILER_FILTER_RANGE.last7days.title}
           testCaseName={testCaseName}
           testCaseParameterValue={entitySpecificInfo?.parameterValues}
-          testCaseResults={entitySpecificInfo?.testCaseResult ?? []}
+          testCaseResults={testCaseResult}
         />
       </Row>
     );
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardV2/FeedCardBody/TestCaseFeed/TestCaseFeed.tsx:60]: Doesn't it require to add `testCaseResult` as dependency in useMemo now?

**Review by aniketkatkar97 — COMMENTED** [review]
@Ashish8689 Left a minor comment

**Review by aniketkatkar97 — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**